src/transformers/models/auto/configuration_auto.py

```diff
@@ -456,6 +458,7 @@
         ("bloom", "BLOOM"),
         ("bort", "BORT"),
         ("bridgetower", "BridgeTower"),
+        ("bros", "Bros"),
```

Comment:
```suggestion
        ("bros", "BROS"),
```
If you run `make fix-copies` after this, it will update it everywhere.

Reply:
Applied suggestions from code review!